ChangeLog.md

```diff
@@ -0,0 +1,42 @@
+## Changes in 1.0.1
+
+* [gui.h](include/clap/ext/gui.h): fix doc: set_scale must be provided
+* [events.h](include/clap/events.h): remove `clap_event_type` which was never used
+* [draft/cv.h](include/clap/ext/draft/cv.h): rename `CLAP_EXT_CV` to `CLAP_EXT_TRANSPORT_CONTROL`
```

Comment:
This was already on main, but after the 1.0.0 tag: https://github.com/free-audio/clap/commit/ba1ed6c663e328e1c495b739c2c894d5eb61fa2e (#89)

Reply:
Those changes were pushed to main after 1.0.0.
See:
039c0f06a2e2b21e6bb54fd9eaa95820e7c85d66
ba1ed6c663e328e1c495b739c2c894d5eb61fa2e
d7ab2d4156407488ee410353fa6072bb09f75f6e

Reply:
Great. But should it say 'rename from' rather than 'rename to'? I see the name now is `CLAP_EXT_CV`.

Other than that all fine! If you make the change I'll submit an approve.